src/libAtomVM/popcorn/popcorn_ets.c

```diff
@@ -735,7 +735,38 @@ PopcornEtsErrorCode popcorn_ets_update_counter(term ref, term key, term operatio
     return insert_result;
 }
 
-PopcornEtsErrorCode popcorn_ets_update_element(term ref, term key, term value, size_t index, term *ret, Context *ctx)
+PopcornEtsErrorCode popcorn_ets_update_element_put_tuple(term element_spec_tuple, term to_insert)
```

Comment:
Name is weird and it should be `static`. Maybe `popcorn_ets_update_element_internal`?